frontend/projects/abtesting/src/assets/i18n/en.json

```diff
@@ -96,6 +97,7 @@
   "home.new-experiment.design.add-experiment-partition.text": "Add Experiment Site",
   "home.new-experiment.design.assignment-weight-validation.text": "Assignment weights' sum must be 100",
   "home.new-experiment.design.condition-unique-validation.text": "Condition should be unique",
+  "home.new-experiment.design.condition-name-validation.text": "Condition name cannot be 'default'",
```

Comment:
Shouldn't message be like **Condition name can not be 'default'**. Space between cannot.